dotCMS/src/main/java/com/dotmarketing/business/VersionableAPI.java

```diff
@@ -295,11 +295,8 @@
 	 * 
 	 * @param ver
 	 * @return
-	 * @throws DotDataException
-	 * @throws DotStateException
-	 * @throws DotSecurityException
 	 */
-	public String getLockedBy(Versionable ver) throws DotDataException, DotStateException,DotSecurityException;
+	public Optional<String> getLockedBy(Versionable ver) throws DotDataException;
```

Comment:
Codacy found an issue: [Unnecessary modifier 'public' on method 'getLockedBy': the method is declared in an interface type](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6214071)